examples/keyvalue/server/observer.go

```diff
@@ -32,7 +32,7 @@ type Observer struct {
 
 // OnInit is called during service init process. Returning an error halts the init?
 func (o *Observer) OnInit(svc service.Host) error {
-	svc.Logger().Info(
+	svc.Logger().Sugar().Infow(
```

Comment:
How do you feel about `svc.TypedLogger()` and `svc.Logger()` being sugared? I'm on the fence as to how we want to expose this, but my gut (and my discussions outside of uber) is that most people find the verbosity of `zap.String()` etc too high, and will opt for typed only when perf is a concern.